python-threatexchange/threatexchange/extensions/vpdq/README.md

```diff
@@ -0,0 +1,79 @@
+# Summary
+The description of vPDQ algorithm hashing and brute-force matching can be found in [readme](../../../../vpdq/README.md). This README describes the [FAISS](https://github.com/facebookresearch/faiss) matching process which is integrated with the vPDQ index in python-threatexchange.
```

Comment:
nit: update link's text to include `vpdq/README.md` for clarity (and incase the link breaks)